src/video/esp-idf/SDL_espidfvideo.c

```diff
@@ -74,11 +74,13 @@ static int ESPIDF_VideoInit(SDL_VideoDevice *_this)
         .max_transfer_sz = (BSP_LCD_H_RES * BSP_LCD_V_RES) * sizeof(uint16_t),
 #endif
     };
-    ESP_ERROR_CHECK(bsp_display_new(&bsp_disp_cfg, &panel_handle, &panel_io_handle));
+    ESP_ERROR_CHECK(bsp_display_new(NULL, &panel_handle, &panel_io_handle));
```

Comment:
Yes, it's my fault, all boards need to be matched here. I am only using this on esp_p4_box now.

Reply:
It might be good to discuss the API with @espzav, so that all GUI vendors do not need to add such ifdef logic. What's your opinion @espzav?
FYI @espressif2022